lib/simple-connect-policy.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+
+// Simple generic connection provider. Used for user-side connection.
+// You are free to implement whatever suits your needs instead.
+// Sends handshake with login/password if provided otherwise sends
+// anonymous handshake.
+//
+module.exports = class SimpleConnectPolicy extends EventEmitter {
```

Comment:
@aqrln yeah, EventEmitter is redundant there, thanks. Auth policy can be just function and I'll allow Connect policy to also be a function as that is indeed more flexible.